src/chat.cpp

```diff
@@ -781,6 +852,28 @@ ULONG CHAT_GetChatMode( void )
 	return ( g_ulChatMode );
 }
 
+//*****************************************************************************
+//
+const char *CHAT_GetChatMessage( ULONG ulPlayer, ULONG ulOffset )
+{
+	FString message = g_SavedChatMessages[ulPlayer].getOldestEntry( ulOffset );
+	return ( message.GetChars( ));
```

Comment:
I think this may break: You return a pointer to the data stored in the `FString message`, but this `FString` leaves the scope once the function is exited, thus it's destroyed and its memory freed. Why don't you return 
`g_SavedChatMessages[ulPlayer].getOldestEntry( ulOffset ).GetChars( )?`

Reply:
Sure thing, I'll change that quickly.